documentation/src/docs/asciidoc/release-notes-5.0.0-M1.adoc

```diff
@@ -23,7 +23,7 @@ on GitHub.
 ** See the https://github.com/junit-team/junit5/labels/up-for-grabs[up-for-grabs] label
    on GitHub.
 * Group IDs, artifact IDs, and versions have changed for all published artifacts.
-** See <<release-notes-5.0.0-m1-migration-artifacts>> and <<dependency-metadata>>.
+** See <<release-notes-5.0.0-m1-migration-artifacts>> and <<index.adoc#dependency-metadata,Dependency Metadata>>.
```

Comment:
Does the cross reference support not pick up the "Dependency Metadata" section header automatically?

Reply:
Unfortunately asciidoctor `cannot(yet) resolve titles automatically`. Please see: http://asciidoctor.org/docs/user-manual/#inter-document-cross-references

Reply:
Ahhhh.... too bad!

I missed that nuance.